WordPress/Classes/ViewRelated/Blog/QuickStartTourGuide.swift

```diff
@@ -317,6 +324,8 @@ private extension QuickStartTourGuide {
         if allToursCompleted(for: blog) {
             WPAnalytics.track(.quickStartAllToursCompleted)
             grantCongratulationsAward(for: blog)
+        } else if Feature.enabled(.quickStartV2) {
+            setLocalNotification(for: blog)
```

Comment:
My feeling is that the notification has to be activated elsewhere. The design says `Quick start is active on the most recently selected site`. I take that to mean we have to turn off the notification when a different blog is selected, and enable it whenever a blog is selected that has QS enabled. 

This is probably destined for `BlogDetailsViewController`?

Reply:
A notification (for the next available task) is created only if a user complete a task. So in theory, if you change site, you loose the previous notification and you won't have a new one till you complete a task. In the Sites list, if the selected blog is different from the current one, I cancel the pending notification.